ydb/core/persqueue/pq_impl.cpp

```diff
@@ -2648,6 +2653,14 @@ void TPersQueue::HandleEventForSupportivePartition(const ui64 responseCookie,
                                                    sender);
         }
     } else {
+        if (!firstWrite) {
+            ReplyError(ctx,
+                       responseCookie,
+                       NPersQueue::NErrorCode::BAD_REQUEST,
+                       "lost messages");
```

Comment:
Code means that client doing smthng wrong. But in this case it os not his fault. PRECONDITION_FAILED or some other code is preferable.

Reply:
Fixed
https://github.com/ydb-platform/ydb/pull/4758/commits/083ea2518891ac980926be5e9c317679380ec934